dev/AppLifecycle/ActivationRegistrationManager.cpp

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#include <pch.h>
+#include <ActivationRegistrationManager.h>
+#include <ActivationRegistrationManager.g.cpp>
+
+#include "LaunchActivatedEventArgs.h"
+#include "ProtocolActivatedEventArgs.h"
+#include "Shared.h"
+
+namespace winrt::Microsoft::ProjectReunion::implementation
+{
+    void ActivationRegistrationManager::RegisterForFileTypeActivation(hstring const& groupName,
+        hstring const& logo, array_view<hstring const> supportedFileTypes,
+        array_view<hstring const> supportedVerbs)
+    {
+        throw hresult_not_implemented();
```

Comment:
I defer to Jon on whether we are allowing "not impls" into main.

Reply:
IIRC there's parts of this overall effort that require additional work, right? Integration with Project Reunion toasting, for instance. I'd _prefer_ there's an implementation, or one coming soon.  We should avoid having a release with this API present but not implemented yet.